### PR TITLE
feat: add perceptor tab

### PR DESCRIPTION
**Pull request by heming6666**
Signed-off-by: LinHaiming <lhming23@outlook.com>

## Purpose
Add perceptor tab.

image

## Proposed changes
_Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue._

#### Open Questions and Pre-Merge TODOs
- [ ] Use github checklists. When solved, check the box and explain the answer.

#### Types of changes
_What types of changes does your code introduce to hypertrons-crx?_
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)

## Approach
_How does this change address the problem?_

## Checklist
_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have read the [CONTRIBUTING](https://github.com/hypertrons/hypertrons-crx/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla-assistant.io/hypertrons/hypertrons-crx)
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

## Further comments (Optional)
_If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc..._
_Links to blog posts, patterns, libraries or addons used to solve this problem_

### CODE REVIEW

**Review by LiuChangFreeman — COMMENTED** [review]
Good Job

File: src/pages/Content/common/DashboardService.tsx [hunk 1]
```diff
@@ -36,7 +40,7 @@ export const renderDashboard = (componentsConf: ComponentConfig[]) => {
 
     const ele = insertElement();
     if (!elementExists(ele)) {
-      console.log('Error! Element not exist!')
+      console.log('Info: Element not exist!')
```
Comment (frank-zsy) [inline src/pages/Content/common/DashboardService.tsx:43]: Do we need a framework logger rather than `console.log`?

Comment (heming6666) [reply src/pages/Content/common/DashboardService.tsx:43]: Thanks @frank-zsy . I think `console.log` is enough for Chrome Extention. Maybe we can also make a simple wrapper for `console.log`, `console.info` and `console.error`.

**Pull request merged**